### PR TITLE
Copy govsvc images to GitHub Container Registry

### PR DESCRIPTION
**Pull request by Krenair**
Where we anticipate fewer rate limiting related issues.

### CODE REVIEW

File: reliability-engineering/pipelines/build-govsvc-docker.yml [hunk 1]
```diff
@@ -6,6 +6,13 @@ resources:
     password: ((dockerhub-password))
     repository: govsvc/aws-ruby
 
+- name: github-aws-ruby
+  type: docker-image
+  source:
+    username: ((github-packages-username))
+    password: ((github-packages-token))
+    repository: ghcr.io/alphagov/aws-ruby
```
Comment (chrisfarms) [inline reliability-engineering/pipelines/build-govsvc-docker.yml:14]: think you are missing a part of the path in these...

should be:

` ghcr.io/alphagov/SOMETHING/aws-ruby`

where SOMETHING = anything you like... maybe "gds" if we think these are global ... or "autom8" or "re" if we think it's important to have a trail back to us

Comment (Krenair) [reply reliability-engineering/pipelines/build-govsvc-docker.yml:14]: That doesn't appear to be necessary. I already created the packages without adding such prefixes.

Comment (chrisfarms) [reply reliability-engineering/pipelines/build-govsvc-docker.yml:14]: TIL 👍

**Review by chrisfarms — APPROVED** [review]

**Review by szd55gds — APPROVED** [review]
LGTM

**Pull request merged**